var/spack/repos/builtin/packages/py-netcdf4/package.py

```diff
@@ -18,15 +18,23 @@ class PyNetcdf4(PythonPackage):
     version('1.4.2',   sha256='b934af350459cf9041bcdf5472e2aa56ed7321c018d918e9f325ec9a1f9d1a30')
     version('1.2.7',   sha256='0c449b60183ee06238a8f9a75de7b0eed3acaa7a374952ff9f1ff06beb8f94ba')
     version('1.2.3.1', sha256='55edd74ef9aabb1f7d1ea3ffbab9c555da2a95632a97f91c0242281dc5eb919f')
-
+    variant(
+        "parallel",
+        default=True,
+        description="Parallel IO support"
+    )
     depends_on('py-setuptools',   type='build')
     depends_on('py-cython@0.19:', type='build')
 
     depends_on('py-numpy@1.7:', type=('build', 'run'))
     depends_on('py-cftime', type=('build', 'run'))
+    depends_on('py-mpi4py', when='+parallel')
```

Comment:
This should probably be `type=('build', 'run')`

Reply:
done